runtime/moonbeam/src/xcm_config.rs

```diff
@@ -712,19 +715,14 @@ impl frame_support::traits::Contains<Location> for AssetFeesFilter {
 	}
 }
 
-pub type AddSupportedAssetOrigin = EitherOfDiverse<
-	EnsureRoot<AccountId>,
-	EitherOfDiverse<
-		pallet_collective::EnsureProportionMoreThan<AccountId, OpenTechCommitteeInstance, 5, 9>,
-		governance::custom_origins::GeneralAdmin,
-	>,
->;
-
-pub type EditSupportedAssetOrigin = EitherOfDiverse<
+pub type AddAndEditSupportedAssetOrigin = EitherOfDiverse<
```

Comment:
Same as above

Reply:
The thing is that we have a different origin for the remove action, calling it manager may be misleading.

Reply:
Fair enough.